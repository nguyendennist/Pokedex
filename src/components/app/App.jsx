import React from 'react';
import { Provider } from 'react-redux';
import store from 'store';
import NextButton from 'containers/next-button/NextButton.jsx';
import PrevButton from 'containers/prev-button/PrevButton.jsx';

const App = () => (
	<Provider store={store}>
		<div>
			<h1>hi!</h1>
			<PrevButton />
			<NextButton />
		</div>
	</Provider>
);

export default App;
