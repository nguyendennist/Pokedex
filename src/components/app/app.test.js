import React from 'react';
import { create } from 'react-test-renderer';
import App from './App.jsx';

describe('unit tests', () => {
	test('should be defined', () => {
		expect(App).toBeDefined();
	});
})

describe('ui tests', () => {
	test('should render correctly', () => {
		const tree = create(
			<App />
		).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
