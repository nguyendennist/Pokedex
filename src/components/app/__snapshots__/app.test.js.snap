// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui tests should render correctly 1`] = `
<div>
  <h1>
    hi!
  </h1>
  <input
    onClick={[Function]}
    type="button"
    value="prev"
  />
  <input
    onClick={[Function]}
    type="button"
    value="next"
  />
</div>
`;
